cmake/core.cmake

```diff
@@ -28,12 +26,12 @@ endif()
 # libraries in our public interface.
 target_link_libraries(mbgl-core PUBLIC
     boost
+    mapbox-base-extras
     mapbox-base
-    geojson.hpp
     ${ICU_LIBRARY}
     polylabel
     protozero
-    rapidjson
+    Mapbox::Base::Extras::rapidjson
```

Comment:
Just curious why rapidjson is in extras? It is mandatory component for code, while, e.g.,  pixelmatch is not.

Reply:
AFAIK `extras` in mapbox-base means just 3rd parties cc @brunoabinader